gradio/components.py

```diff
@@ -4127,7 +4127,17 @@ def __init__(
         )
 
     def get_config(self):
-        return {"value": self.value, **IOComponent.get_config(self)}
+        try:
+            import bokeh
+
+            bokeh_version = bokeh.__version__
+        except ImportError:
+            bokeh_version = "3.0.3"
```

Comment:
It doesn't matter which version is the "fallback" version since there can't be a bokeh plot if bokeh is not installed so it's not like the front-end and backend bokeh versions would be out of synch.

Reply:
Why do we need a fallback version anyways as opposed to being `None`?

Reply:
I should test this but my understanding was that the front end always loads the bokeh assets from the cdn, in which case we None wouldn’t work

Reply:
Ah I see, but maybe we don't have to import the bokeh assets at all if bokeh is not installed?